sdk/storage/azure-storage-file-share/src/main/java/com/azure/storage/file/share/models/CopyableFileSmbPropertiesList.java

```diff
@@ -100,7 +101,7 @@ public CopyableFileSmbPropertiesList setChangedOn(boolean changedOn) {
     /**
      * @return a flag indicating whether all attributes should be copied from source file.
      */
-    public boolean getAll() {
+    public Boolean getAll() {
```

Comment:
This needs changed to `isAll()` per this API view - https://apiview.dev/Assemblies/Conversation/a5d1fc93e194440c885e6cc8df4c2e7a